src/main/navigation/navigation_pos_estimator.c

```diff
@@ -385,7 +388,11 @@ static void updateIMUTopic(timeUs_t currentTimeUs)
     }
     else {
         /* Update acceleration weight based on vibration levels and clipping */
-        updateIMUEstimationWeight(dt);
+        if (positionEstimationConfig()->dynamic_acc_weight) {
```

Comment:
Not sure it this is needed. And if it is, it should be hard coded to 0.3f when off.